test/Conversion/TorchOnnxToTorch/simple_ops_a_to_f.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: torch-mlir-opt <%s --split-input-file -convert-torch-onnx-to-torch | FileCheck %s
+// RUN: torch-mlir-opt <%s --split-input-file -convert-torch-onnx-to-torch --torch-decompose-complex-ops --canonicalize --torch-backend-to-linalg-on-tensors-backend-pipeline
```

Comment:
What is this actually testing, without piping to FileCheck?

Rather than splice the files this way and have them tested through two different pipelines, how about forking the test cases into a new `TorchOnnxToLinalg` folder where linalg can be tested directly?

(Alternately keep the test cases in a single file but use `FileCheck -check-prefix=CHECK-LINALG` to differentiate between the different `RUN` commands)

Reply:
@kumardeepakamd @rsuderman Would you like to add something here?